pkg/liqoctl/install/provider/generic.go

```diff
@@ -37,6 +38,16 @@ func (p *GenericProvider) ValidateGenericCommandArguments(flags *flag.FlagSet) (
 		return err
 	}
 
+	if clusterLabelsFlag := flags.Lookup("cluster-labels"); clusterLabelsFlag != nil {
```

Comment:
the label name should be a constant (should be already available somewhere)

Reply:
I didn't find it anywhere, however I added it at the top of the file. Other flags are specified as hard-coded values, but I could also find a constant for "cluster-name" already specified in pkg/const/parameters.go. Should I add the new constants in that file, instead of specifying them at the top of this file?